library/Vanilla/ApiUtils.php

```diff
@@ -157,6 +157,7 @@ private static function pagerUrlFormat($url, array $query, Schema $schema) {
         if (!empty($argsStr)) {
             $url .= (strpos($url, '?') === false ? '?' : '&').$argsStr;
         }
+        $url = \Gdn::request()->url($url, true);
```

Comment:
This may end up adding double subcommunity slugs onto the URLs. (Best to always test URL handling in a sub-folder, with subcommunities enabled).

`$request->getSimpleUrl($url)` is probably what you're looking for.